### PR TITLE
Case Access Management Release

### PR DESCRIPTION
**Pull request by ss-evoco**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by damien-cooke — APPROVED** [review]

**Review by TeaWithLucas — APPROVED** [review]

File: benefit/data/sheets/CaseTypeTab/CaseTypeTab-nonprod.json [hunk 1]
```diff
@@ -1,11 +1,4 @@
...
-  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "caseNameHmctsRestricted", "TabFieldDisplayOrder": 2},
-  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "caseNamePublic", "TabFieldDisplayOrder": 3},
-  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "ogdType", "TabFieldDisplayOrder": 4},
-  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "caseManagementCategory", "TabFieldDisplayOrder": 5},
-  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "CaseAccessCategory", "TabFieldDisplayOrder": 6},
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "SearchCriteria", "TabFieldDisplayOrder": 7},
```
Comment (TeaWithLucas) [inline benefit/data/sheets/CaseTypeTab/CaseTypeTab-nonprod.json:3]: trailing ,

File: benefit/data/sheets/CaseTypeTab/CaseTypeTab.json [hunk 2]
```diff
@@ -265,5 +266,12 @@
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "language_sscs", "TabLabel": "Welsh", "TabDisplayOrder": 25, "CaseFieldID": "languagePreferenceWelsh", "TabFieldDisplayOrder": 1, "TabShowCondition": "languagePreferenceWelsh=\"Yes\""},
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "language_sscs", "TabLabel": "Welsh", "TabDisplayOrder": 26, "CaseFieldID": "translationWorkOutstanding", "TabFieldDisplayOrder": 2},
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "hearingRecordingsInfo_sscs", "TabLabel": "Hearing Recordings", "TabDisplayOrder": 27, "CaseFieldID": "sscsHearingRecordings", "TabFieldDisplayOrder": 2},
-  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "other_party_sscs", "TabLabel": "Other Party Details", "TabDisplayOrder": 28, "CaseFieldID": "otherParties", "TabFieldDisplayOrder": 1}
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "other_party_sscs", "TabLabel": "Other Party Details", "TabDisplayOrder": 28, "CaseFieldID": "otherParties", "TabFieldDisplayOrder": 1},
+
```
Comment (TeaWithLucas) [inline benefit/data/sheets/CaseTypeTab/CaseTypeTab.json:270]: extra whitespace

File: benefit/data/sheets/CaseTypeTab/CaseTypeTab.json [hunk 3]
```diff
@@ -265,5 +266,12 @@
...
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "TabID": "wa_fields", "TabLabel": "WA fields", "TabDisplayOrder": 29, "CaseFieldID": "caseNameHmctsInternal", "TabFieldDisplayOrder": 1},
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "caseNameHmctsRestricted", "TabFieldDisplayOrder": 2},
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "caseNamePublic", "TabFieldDisplayOrder": 3},
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "ogdType", "TabFieldDisplayOrder": 4},
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "caseManagementCategory", "TabFieldDisplayOrder": 5},
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "CaseAccessCategory", "TabFieldDisplayOrder": 6}
```
Comment (TeaWithLucas) [inline benefit/data/sheets/CaseTypeTab/CaseTypeTab.json:276]: The tab label is "Appeal Details", but there is already a tab called that, but the tab id is wa_fields.
Is this meant to be a separate tab?

File: benefit/data/sheets/CaseTypeTab/CaseTypeTab.json [hunk 4]
```diff
@@ -265,5 +266,12 @@
...
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "language_sscs", "TabLabel": "Welsh", "TabDisplayOrder": 26, "CaseFieldID": "translationWorkOutstanding", "TabFieldDisplayOrder": 2},
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "hearingRecordingsInfo_sscs", "TabLabel": "Hearing Recordings", "TabDisplayOrder": 27, "CaseFieldID": "sscsHearingRecordings", "TabFieldDisplayOrder": 2},
-  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "other_party_sscs", "TabLabel": "Other Party Details", "TabDisplayOrder": 28, "CaseFieldID": "otherParties", "TabFieldDisplayOrder": 1}
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "other_party_sscs", "TabLabel": "Other Party Details", "TabDisplayOrder": 28, "CaseFieldID": "otherParties", "TabFieldDisplayOrder": 1},
+
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "TabID": "wa_fields", "TabLabel": "WA fields", "TabDisplayOrder": 29, "CaseFieldID": "caseNameHmctsInternal", "TabFieldDisplayOrder": 1},
```
Comment (TeaWithLucas) [inline benefit/data/sheets/CaseTypeTab/CaseTypeTab.json:271]: Are the following fields only for superuser?

File: benefit/data/sheets/CaseEvent/CaseEvent.json [hunk 5]
```diff
@@ -2535,5 +2535,16 @@
...
+  {
+    "LiveFrom": "01/01/2018",
+    "CaseTypeID": "Benefit",
+    "ID": "migrateCase",
+    "Name": "Migrate Case",
+    "Description": "Migrate Case",
+    "DisplayOrder": 996,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public"
```
Comment (TeaWithLucas) [inline benefit/data/sheets/CaseEvent/CaseEvent.json:2548]: Can you put this in the correct order?

File: benefit/data/sheets/CaseTypeTab/CaseTypeTab-nonprod.json (hunk 1 above)
Comment (TeaWithLucas) [inline benefit/data/sheets/CaseTypeTab/CaseTypeTab-nonprod.json:3]: Why is this field not being added to prod, is it needed?

File: benefit/data/sheets/CaseTypeTab/CaseTypeTab.json (hunk 4 above)
Comment (ss-evoco) [reply benefit/data/sheets/CaseTypeTab/CaseTypeTab.json:271]: Yes

File: benefit/data/sheets/CaseTypeTab/CaseTypeTab.json (hunk 3 above)
Comment (ss-evoco) [reply benefit/data/sheets/CaseTypeTab/CaseTypeTab.json:276]: Will add this as part of CAM

**Review by TeaWithLucas — APPROVED** [review]

**Pull request merged**